pages/patterns/create-a-profile/race-and-ethnicity.md

```diff
@@ -97,9 +100,63 @@ Allow selection of more than one race and acknowledge people with multiple racia
   </h3>
   <div id="accordion-code" class="usa-accordion__content highlight-code copy-code">
     <div class="usa-sr-only">
-      {% highlight html%}{% include patterns/race-and-ethnicity.html %}{% endhighlight %}
+      {% highlight html%}{% include patterns/race-and-ethnicity-wrapper.html content="patterns/race-and-ethnicity-1.html" hint-input=true %}{% endhighlight %}
+    </div>
+    {% highlight html%}{% include patterns/race-and-ethnicity-wrapper.html content="patterns/race-and-ethnicity-1.html" hint-input=true %}{% endhighlight %}
+  </div>
+</div>
+
+{:.site-component-section-title}
+## Some details
+
+<div class="usa-accordion usa-accordion--bordered site-accordion-code site-component-preview margin-top-2">
+  <h3 id="pattern-preview-2" class="usa-accordion__heading site-accordion-heading">
+    <button type="button" class="usa-accordion__button" aria-controls="accordion-preview-2" aria-expanded="true">
+      Pattern preview
+    </button>
+  </h3>
+  <div id="accordion-preview-2" class="usa-accordion__content">
+        {% include patterns/race-and-ethnicity-wrapper.html content="patterns/race-and-ethnicity-2.html" %}
+  </div>
+</div>
+<div class="usa-accordion usa-accordion--bordered site-accordion-code site-component-preview">
+  <h3 id="pattern-code-2" class="usa-accordion__heading site-accordion-heading">
+    <button type="button" class="usa-accordion__button" aria-controls="accordion-code-2" aria-expanded="false">
+      Pattern code
+    </button>
+  </h3>
+  <div id="accordion-code-2" class="usa-accordion__content highlight-code copy-code">
+    <div class="usa-sr-only">
+      {% highlight html%}{% include patterns/race-and-ethnicity-wrapper.html content="patterns/race-and-ethnicity-2.html" %}{% endhighlight %}
+    </div>
+    {% highlight html%}{% include patterns/race-and-ethnicity-wrapper.html content="patterns/race-and-ethnicity-2.html" %}{% endhighlight %}
```

Comment:
## SR only code

Comparing to other code previews with `usa-sr-only`, we use `highlight text` instead of highlight `html`.

Listening and comparing the two, this allows the code block to be read out in it's entirety for SR users instead of tag-by-tag

Can you update in each of the `usa-sr-only` code highlight blocks?

```suggestion
    <div class="usa-sr-only">
      {% highlight text%}{% include patterns/race-and-ethnicity-wrapper.html content="patterns/race-and-ethnicity-2.html" %}{% endhighlight %}
    </div>
    {% highlight html%}{% include patterns/race-and-ethnicity-wrapper.html content="patterns/race-and-ethnicity-2.html" %}{% endhighlight %}
```

Reply:
Good catch! Updated in [cb149d0](https://github.com/uswds/uswds-site/pull/2657/commits/cb149d083ed5b827db4616281e9d348e548a29df)